apps/myaccount/src/init/app-utils.ts

```diff
@@ -73,6 +73,7 @@ export const AppUtils: AppUtilsInterface = (function() {
     const tenantResolutionStrategyFallback: string = "id_token";
     const proxyContextPathFallback: string = "";
 
+    const SERVER_ORIGIN_PLACEHOLDER: string = "${serverOrigin}";
```

Comment:
Shall we use `SERVER_ORIGIN_IDP_URL_PLACEHOLDER` for this since it has same value and its being used other places as well.

Reply:
Fixed with add793a96f08e8044defdfa2499a46eb1ca207f9